### ISSUE TITLE
`deno bench` attempts to parse files read through `Deno.readFile(Sync)`

### ISSUE THREAD

**Issue by aapoalas** [denoland/deno#15154]
Steps to reproduce:
1. Check out the SheetJS Deno repo from https://github.com/SheetJS/nodeno/
2. Change the `read.ts` file to use `Deno.bench()` around the `XLSX.read()` call.
3. Try to run the bench using `deno bench`

Expected result:
1. Bench will work, as run did (well, there're some issues here with V8 crashing but lets not worry about the small stuff)

Actual result:
1. Bench fails with the following error:
```
deno bench --unstable --allow-read=./ --check --allow-env=NODE_DEBUG read.ts large_strings.xlsx.xml
error: The module's source code could not be parsed: Unexpected token `?`. Expected this, import, async, function, [ for array literal, { for object literal, @ for decorator, function, class, null, true, false, number, bigint, string, regexp, ` for template literal, (, or an identifier at file:///home/user/build/nodeno/large_strings.xlsx.xml:1:2
```

The `large_strings.xlsx.xml` file indeed starts like this:
```xml
<?xml version="1.0"?>
```
so the line 1, colon 2 for token "?" is very much correct.

It seems that bench is trying to read the `readFileSync` source file as JS.

Switching to `await Deno.readFile()` does not fix the issue.

**Comment by dsherret** [comment]
I don't think this is a bug because in that position are the files to run for benchmarking:

![image](https://user-images.githubusercontent.com/1609021/178506790-8f2ed778-1ee7-470a-a998-e9d1dfe57b45.png)

I think probably you mean to add a `--` before `large_strings.xlsx.xml`?
